test_requirements/requirements_base.txt

```
@@ -2,7 +2,7 @@ coverage>=4
 python-coveralls>2.5.0
 unittest-xml-reporting==1.11.0
 Pillow==5.2.0
-django-treebeard>=4.3
+django-treebeard<4.5
```

This should match the setup.py entry:

```suggestion
django-treebeard>=4.3,<4.5
```